projects/gmsl-uc/Makefile

```diff
@@ -0,0 +1,10 @@
+# Select the example you want to enable by choosing y and disable others by choosing n.
+
+CFG_DIAG_MAX96792_MAX96793_UC_EX = y
```

Comment:
please use the new `EXAMPLE=max96792_max96793_example` infrastructure

Reply:
click on any commit of this PR to see what you need to change:
https://github.com/analogdevicesinc/no-OS/pull/2420

Reply:
@asewa108 this is another example project that is following this structure. #2459. You can also follow how the commits subjects and body were created. You can use git rebase -i and edit.